### PR TITLE
baklava: fix #2094 and parse time properly

### LINKED ISSUE #2094: Windows 10 notification thinks app name is "electron.app.Dogehouse"
image
It should probably just say "Dogehouse" instead of "electron.app.Dogehouse"

### PR DESCRIPTION
**Pull request by amitojsingh366**
fixes #2094 and shows proper time on RPC

### CODE REVIEW

File: baklava/package.json [hunk 1]
```diff
@@ -129,4 +129,4 @@
         "./icons/**/*",
         "./resources/**/*"
     ]
-}
+}
```
Comment (goldyydev) [inline baklava/package.json:132]: 0/10 PR, No newline

```suggestion
}

```

**Review by goldyydev — CHANGES_REQUESTED** [review]

File: baklava/package.json (hunk 1 above)
Comment (BronzW) [reply baklava/package.json:132]: #GoldyLovesEmNewlines

Comment (amitojsingh366) [reply baklava/package.json:132]: idk why but my editor removes newlines

Comment (BronzW) [reply baklava/package.json:132]: You can turn off all formatting on save in VSCode

Comment (amitojsingh366) [reply baklava/package.json:132]: but i want auto format, i just dont want this

**Pull request merged**